script.js

```diff
@@ -0,0 +1,65 @@
+let bookList = [];
+const awesomeBook = document.getElementById('awesome-book');
+const bttn = document.getElementById('bttn');
+awesomeBook.style.display = 'none';
+
+function displayBooks(list) {
+  let tr = '';
+  let sn = 1;
+  list.forEach((item) => {
+    const tr1 = `<tr>
+    <td>${sn}</td>
+    <td>${item.title}</td>
+    <td>${item.author}</td>
+    <td> <button type ="button" class = "remove-book" data-bookId="${(sn - 1)}">Remove</button></td>
+    </tr>`;
+    tr += tr1;
+    sn += 1;
+  });
+  document.getElementById('tbody').innerHTML = tr;
+}
+
+function getLocalStorageData() {
+  const data = JSON.parse(localStorage.getItem('bookdata'));
+  if (data != null && data.length > 0) {
+    awesomeBook.style.display = 'block';
+    displayBooks(data);
+  } else {
+    awesomeBook.style.display = 'none';
+  }
+}
+
+bttn.onclick = function () {
+  const title = document.getElementById('title');
+  const author = document.getElementById('author');
+  const data = JSON.parse(localStorage.getItem('bookdata'));
+  if (data != null && data.length > 0) {
+    bookList = data;
+  }
+
+  const data2 = {
+    title: title.value,
+    author: author.value,
+  };
+  bookList.push(data2);
+  localStorage.setItem('bookdata', JSON.stringify(bookList));
+  document.getElementById('book-form').submit();
+};
```

Comment:
- [ ] Kindly after pushing the browser is frozen so to get data the user must refresh the page again however the data is stored in the local-storage as you can see at the screenshot bellow  . Fix that by adding a function to refresh the page automatically
Your Project
![Screenshot from 2022-06-04 15-04-40](https://user-images.githubusercontent.com/77842237/171998202-4fc45a69-6bad-4a33-80c5-8d3757d42fc7.png)